pkgs/development/python-modules/spyder/default.nix

```diff
@@ -66,6 +68,7 @@ buildPythonPackage rec {
     license = licenses.mit;
     platforms = platforms.linux;
     maintainers = with maintainers; [ gebner ];
-    broken = true;
+    broken = isPy27;
+    # UnicodeEncodeError: 'ascii' codec can't encode character u'\u2014' in position 15: ordinal not in range(128)
```

Comment:
disabled = ...

I don't think it is supported.

Reply:
Disabled.